<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>843</width>
        <height>631</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color:#000000;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QFrame" name="navbar">
         <property name="minimumSize">
          <size>
           <width>843</width>
           <height>71</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color:#000000;
}

QPushButton{
border:none;
background:none;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="menu">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>img/menu.png</normaloff>img/menu.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>45</width>
                <height>45</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cerrar">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>img/cerrar.png</normaloff>img/cerrar.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main">
         <property name="minimumSize">
          <size>
           <width>843</width>
           <height>631</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color:#000000;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="sidebar">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color:rgb(0, 212, 255);
}

QPushButton{
background-color:gray;
border:none;
color:white;
font-size:18px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="btn_registrar">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>Registrar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_todo">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>Inventario</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_registrarRes">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>Agregar Restaurante</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_todoRes">
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>45</height>
                </size>
               </property>
               <property name="text">
                <string>Restaurantes</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="paginas">
            <property name="minimumSize">
             <size>
              <width>561</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="styleSheet">
                <string notr="true">QWidget{
background-color:#000000;
}

QLineEdit{
border:1px solid #fff;
background-color:#b1b1b1;
}

QComboBox{
border:1px solid #fff;
background-color:#b1b1b1;
}

QLabel{
background:none;
color:#fff;
font-size:16px;
}

QPushButton{
background-color:rgb(0, 212, 255);
border:none;
color:#000000;
border-radius:10px;
}</string>
               </property>
               <property name="currentIndex">
                <number>2</number>
               </property>
               <widget class="QWidget" name="pagina_registrar">
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Registrar Producto</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_12">
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Producto</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_producto">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>41</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_13">
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Cantidad</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_cantidad">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>41</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_14">
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Restaurante</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="combo">
                     <property name="minimumSize">
                      <size>
                       <width>301</width>
                       <height>41</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_agregar">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Agregar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pagina_todo">
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>Inventario</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tabla">
                   <column>
                    <property name="text">
                     <string>Producto</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Cantidad</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>Restaurante</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pagina_todoRes">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Restaurantes</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tablaRes">
                   <column>
                    <property name="text">
                     <string>nombre</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pagina_registrarRes">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Registrar Restaurante</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_4">
                     <item>
                      <spacer name="verticalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Nombre</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="input_nombreRes">
                       <property name="minimumSize">
                        <size>
                         <width>301</width>
                         <height>41</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_agregarRes">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Agregar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
